src/SourceBuild/content/repo-projects/Directory.Build.props

```diff
@@ -1,4 +1,4 @@
-<Project>
+<Project TreatAsLocalProperty="UseOfficialBuildVersioning">
```

Comment:
I would rather not pass `/p:UseOfficialBuildVersioning=true` in from the build scripts when --source-only is true. Or to rephrase, the argument should already be ignored in the build scripts where that behavior is documented instead of within msbuild.